core/jobs/batch_jobs/suggestion_stats_computation_jobs.py

```diff
@@ -64,29 +65,80 @@ class TranslationContributionsStatsDict(TypedDict):
 
 
 class GenerateTranslationContributionStatsJob(base_jobs.JobBase):
-    """Job that indexes the explorations in Elastic Search."""
+    """Job that generates contributor stats."""
 
     def run(self) -> beam.PCollection[job_run_result.JobRunResult]:
-        """Generates the translation contributins stats.
+        """Generates the translation review stats.
 
         Returns:
             PCollection. A PCollection of 'SUCCESS x' results, where x is
             the number of generated stats..
         """
-        suggestions_grouped_by_target = (
+        non_deleted_suggestion_models = (
             self.pipeline
             | 'Get all non-deleted suggestion models' >> ndb_io.GetModels(
                 suggestion_models.GeneralSuggestionModel.get_all(
                     include_deleted=False))
+        )
+
+        translation_suggestions_grouped_by_target = (
+            non_deleted_suggestion_models
             | 'Filter translate suggestions' >> beam.Filter(
                 lambda m: (
                     m.suggestion_type ==
                     feconf.SUGGESTION_TYPE_TRANSLATE_CONTENT
                 ))
-            | 'Transform to suggestion domain object' >> beam.Map(
+            | 'Transform to submitted suggestion domain object' >> (
+                beam.Map(suggestion_services.get_suggestion_from_model))
+            | 'Group submitted suggestions by target' >> (
+                beam.GroupBy(lambda m: m.target_id))
+        )
+        reviewed_translation_suggestions_grouped_by_target = (
+            non_deleted_suggestion_models
+            | 'Filter reviewed translate suggestions' >> beam.Filter(
+                lambda m: (
+                    m.suggestion_type ==
+                    feconf.SUGGESTION_TYPE_TRANSLATE_CONTENT
+                    in [
+                        m.status == suggestion_models.STATUS_ACCEPTED,
+                        m.status == suggestion_models.STATUS_REJECTED
+                    ]
+                ))
+            | 'Transform to reviewed suggestion domain object' >> beam.Map(
                 suggestion_services.get_suggestion_from_model)
-            | 'Group by target' >> beam.GroupBy(lambda m: m.target_id)
+            | 'Group reviewed suggestions by target' >> (
+                beam.GroupBy(lambda m: m.target_id))
+        )
+        question_suggestions_grouped_by_target = (
+            non_deleted_suggestion_models
+            | 'Filter question suggestions' >> beam.Filter(
+                lambda m: (
+                    m.suggestion_type ==
+                    feconf.SUGGESTION_TYPE_ADD_QUESTION
+                ))
+            | 'Transform to submitted question suggestion domain object' >> (
+                beam.Map(suggestion_services.get_suggestion_from_model))
+            | 'Group submitted question suggestions by target' >> (
+                beam.GroupBy(lambda m: m.target_id))
         )
+        question_reviews_grouped_by_target = (
```

Comment:
```suggestion
        reviewed_question_suggestions_grouped_by_target = (
```

For consistency and clarity.

Reply:
Done